README.md

```diff
@@ -36,6 +38,8 @@ server:
   serveStatic:
     extensions:
     - html
+  proxyPath: undefined
+  proxyUrl: undefined
```

Comment:
Conventionally `http://example.com` is of documental usage. 

Also for the sake of simplicity I suggest simply pass on the option to http-proxy-middleware directly so people can make most use of the potential.

The following suggestion also remove the `proxyPath: undefined` settings.
```suggestion
  httpProxy:
    context: /foo/bar
    config:
      target: http://example.com
      changeOrigin: true
```

Here the naming of `httpProxy` corresponds to `http-proxy-middleware`, `context`, `config` to the signature in the document [here](https://github.com/chimurai/http-proxy-middleware#proxycontext-config).